src/ratbag_emu/protocol/util/profile.py

```diff
@@ -1,47 +1,89 @@
 from collections import namedtuple
 
+
 '''
 Imeplemnts a profile with all the mouse settings
 '''
 class Profile(object):
     Led = namedtuple('Led', ['red', 'green', 'blue'])
 
-    def __init__(self, obj=None):
+    class Button(object):
```

Comment:
is there a specific need to have a nested class?

Reply:
I can move it out. I just did it to keep in sync with `Led` which was already there.

Reply:
OK then